tests/sanity/ignore.txt

```diff
@@ -550,3 +550,9 @@ test/units/utils/test_shlex.py future-import-boilerplate
 test/units/utils/test_shlex.py metaclass-boilerplate
 test/utils/shippable/check_matrix.py replace-urlopen
 test/utils/shippable/timing.py shebang
+plugins/modules/nios_a_record.py:0:0: doc-required-mismatch: Argument 'host' in argument_spec found in provider is not required, but is documented as being required
```

Comment:
```
plugins/modules/nios_a_record.py validate-modules:doc-required-mismatch
```
It should look like above for the selected test, can you make the changes and do a re-run

Reply:
Done but it didn't reduce the no of sanity errors.

Reply:
ahh, just saw this, let me check this on my end and confirm